paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -129,7 +132,7 @@ class Paparazzi(
         .copy(
             layoutPullParser = LayoutPullParser.createFromString(contentRoot),
             deviceConfig = deviceConfig,
-            renderingMode = SessionParams.RenderingMode.V_SCROLL
+            renderingMode = SessionParams.RenderingMode.NORMAL
```

Comment:
this didn't work until layoutlib 4.2, and now results infinitely scrolling lists, which may not be desirable by default.  cc: @gamepro65 who has a pending fix to expose this configuration to test authors.